var/spack/repos/builtin/packages/nco/package.py

```diff
@@ -12,17 +12,38 @@ class Nco(AutotoolsPackage):
     homepage = "https://nco.sourceforge.net/"
     url = "https://github.com/nco/nco/archive/5.0.1.tar.gz"
 
+    maintainers("altheaden", "xylar")
+
     license("BSD-3-Clause")
 
+    version("5.3.1", sha256="c527e991e1befcc839a14151a2982a20340ab1523ce98b66ef3efa2878ee039b")
```

Comment:
@altheaden, can you also add 5.3.2?
https://github.com/E3SM-Project/spack/commit/3c0c9ae01eaededa40cf2281939f23dbe65a8e43